src/main/java/core/basesyntax/service/transaction/impl/CsvFruitTransactionParsingService.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.transaction.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.transaction.FruitTransactionParsingService;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CsvFruitTransactionParsingService implements FruitTransactionParsingService {
+    private static final String CSV_SEPARATOR = ",";
+    private static final int FRUIT_TRANSACTION_OPERATION_INDEX = 0;
+    private static final int FRUIT_TRANSACTION_FRUIT_INDEX = 1;
+    private static final int FRUIT_TRANSACTION_QUANTITY_INDEX = 2;
```

Comment:
Do not complicate names
```suggestion
    private static final int OPERATION_INDEX = 0;
    private static final int FRUIT_INDEX = 1;
    private static final int QUANTITY_INDEX = 2;
```